Monika After Story/game/zz_calendar.rpy

```diff
@@ -1763,7 +1763,52 @@ init python:
             []
         )
 
-    # TODO: add first kiss here
+    # add first kiss
+    if (
+            persistent._mas_first_kiss is not None
+            and type(persistent._mas_first_kiss) == datetime.datetime
+        ):
+        calendar.addRepeatable_dt(
+            "first-kiss",
+            "Our First Kiss",
+            persistent._mas_first_kiss,
+            []
+        )
+
+# Using init 2 so we can have access to the season dates
+init 2 python:
+    import store.mas_calendar as calendar
+
+    # Season changes:
+    if persistent._mas_pm_live_south_hemisphere:
```

Comment:
make this into a function that we can call from the hemispheres topic to update the names.

And it might be beneficial to add a `changeRepeatableName` function to calendar module that can be used to change the display names of repeatables without having to remove and readd.